src/components/PrimaryNav/PrimaryNav.tsx

```diff
@@ -103,6 +100,11 @@ const PrimaryNav = () => {
         <li className="p-list__item">
           <ControllersLink />
         </li>
+        {!isJuju && (
```

Comment:
As we are adding the Cross Model Search in JIMM, I thought that the user shouldn't be able to access this link on the sidebar when using a Juju controller. Let me know if this isn't the case.

Reply:
Good catch!